### ISSUE TITLE
undefined method `delegate' for class #<Class:ShopifyGraphql> 

### ISSUE THREAD

**Issue by thomaswitt** [kirillplatonov/shopify_graphql#40]
Wanted to try the gem and installed it, then calling it straight from irb, getting the following error:

```
thomas@Mac:~/Dev/misc/ $ ruby -v
ruby 3.3.1 (2024-04-23 revision c56cd86388) [arm64-darwin23]
thomas@Mac:~/Dev/misc/ $ irb
>> require 'shopify_graphql'
/Users/thomas/.rbenv/versions/3.3.1/lib/ruby/gems/3.3.0/gems/shopify_graphql-1.2.5/lib/shopify_graphql/client.rb:92:in `singleton class': undefined method `delegate' for class #<Class:ShopifyGraphql> (NoMethodError)

    delegate :execute, :handle_user_errors, to: :client
    ^^^^^^^^
Did you mean?  DelegateClass
	from /Users/thomas/.rbenv/versions/3.3.1/lib/ruby/gems/3.3.0/gems/shopify_graphql-1.2.5/lib/shopify_graphql/client.rb:91:in `<module:ShopifyGraphql>'
```

**Comment by kirillplatonov** [comment]
I will take a look at that. For now you can try it by adding gem into Rails app and calling it from `rails console`.

**Comment by thomaswitt** [comment]
@kirillplatonov has this been fixed in a commit or is it just closed?

**Comment by kirillplatonov** [comment]
@thomaswitt Fixed in #48. I'm gonna publish new version shortly.
